product-service/handlers/getProduct.ts

```diff
@@ -0,0 +1,31 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import productList from '../productList.json';
+
+export const getProduct: APIGatewayProxyHandler = async (event, _context) => {
+	const { productId } = event.pathParameters;
+	const product = productList.find(
+		({ id }) => event.pathParameters.productId === id
```

Comment:
You should use `productId` variable instead of `event.pathParameters.productId`

Reply:
Done